scripts/export-rmib-onnx.py

```
@@ -0,0 +1,215 @@
+
+def preprocess(bvh_dir):
+    from bvh import Bvh
+    files = sorted(glob.glob(os.path.join(bvh_dir, "**/*.bvh"), recursive=True))
+    print(f"found {len(files)} BVH files")
+    S, E, I = [], [], []
+    for i, path in enumerate(files):
+        try:
+            with open(path) as f:
+                m = Bvh(f.read())
+        except Exception:
+            continue
```

Ruff (0.15.18)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)